### PR TITLE
implement missing esri geometry types

### PR DESCRIPTION
**Pull request by mki-c2c**
Some esri geometry types were not implemented.

geometry parsing has been implemented manually, since there is no known library for conversion from ArcGIS feature format to geoJSON.

The app can now handle the following geometry types:
['esriGeometryPoint', 'esriGeometryMultipoint', 'esriGeometryPath', 'esriGeometryPolyline', 'esriGeometryRing', 'esriGeometryPolygon', 'esriGeometryEnvelope']

### CODE REVIEW

**Comment by mki-c2c** [comment]
... still the same: untested except for 'pistes cyclables nationales' layer with Polyline geometries, correct behaviour for other geometries has to be checked on nextprod

File: geoportal/geoportailv3_geoportal/views/fulltextsearch.py [hunk 1]
```diff
@@ -445,23 +445,23 @@ def featuresearch(self, layers=None):
                     return []
 
                 try:
-                    esricoll = geojson.loads(content)
+                    log.info(fiona.__version__)
+                    log.info(fiona.get_gdal_release_name())
+                    log.info(fiona.supported_drivers)
```
Comment (tonio) [inline geoportal/geoportailv3_geoportal/views/fulltextsearch.py:450]: that could be deleted now.

File: geoportal/geoportailv3_geoportal/views/fulltextsearch.py [hunk 2]
```diff
@@ -445,23 +445,23 @@ def featuresearch(self, layers=None):
...
+                    mf = fiona.MemoryFile(content)
+                    esricoll = mf.open(driver='ESRIJSON')
```
Comment (tonio) [inline geoportal/geoportailv3_geoportal/views/fulltextsearch.py:452]: shoudn't it be closed ? (or use with a context manager)

**Review by tonio — APPROVED** [review]
Looks good now, better to rely on more generic libraries. Thanks!

Just a few minor comments to fix, please.

File: geoportal/geoportailv3_geoportal/views/fulltextsearch.py (hunk 1 above)
Comment (mki-c2c) [reply geoportal/geoportailv3_geoportal/views/fulltextsearch.py:450]: oops

File: geoportal/geoportailv3_geoportal/views/fulltextsearch.py (hunk 2 above)
Comment (mki-c2c) [reply geoportal/geoportailv3_geoportal/views/fulltextsearch.py:452]: yes indeed.

**Pull request merged**